docs/authorization/access-policies-guide.md

```diff
@@ -0,0 +1,312 @@
+# About DataHub Access Policies
+
+<FeatureAvailability/>
+
+At the lowest level of DataHub's access control model is the concept of **Access Policies**. In conjunction with [Roles](./roles.md), Access Policies determine what users are allowed to do on DataHub.
+More specifically, Access Policies define *who* can *do what* to *which resources*.
+
+A few policies in plain English include
+
+- Dataset Owners should be allowed to edit documentation, but not Tags.
+- Jenny, our Data Steward, should be allowed to edit Tags for any Dashboard, but no other metadata.
+- James, a Data Analyst, should be allowed to edit the Links for a specific Data Pipeline he is a downstream consumer of.
+- The Data Platform team should be allowed to manage users & groups, view platform analytics, & manage policies themselves.
+
+Each of these can be implemented by constructing DataHub Access Policies.
+
+In this guide, we'll describe how to create & manage Access Policies.
+
+## Access Policies Setup, Prerequisites, and Permissions
+
+What you need to manage Access Policies on DataHub:
+
+* **Manage Policies** privilege
+
+This platform privilege allows users to create, edit, and remove all Access Policies on DataHub. Therefore, it should only be
+given to those users who will be serving as Admins of the platform. The default `Admin` role has this privilege.
+
+
+## Using Access Policies
+
+### Creating a new Policy
+
+Policies can be created by first navigating to **Settings > Permissions > Policies**. The `Policies` tab will only
+be visible to those users having the required `Manage Policies` privilege.
+
+To begin building a new Policy, click **Create new Policy**.
+
+<p align="center">
+  <img width="40%"  src="https://raw.githubusercontent.com/datahub-project/datahub/master/docs/imgs/manage-permissions.png"/>
+</p>
+
+#### Step 1. Choose Policy Type
+
+In the first step, we'll give our Policy a name and description. We'll also select the *type* of the policy,
+which determines which privileges can be granted using it.
+
+First give the new Policy a name.
+
+<p align="center">
+  <img width="40%"  src="https://raw.githubusercontent.com/datahub-project/datahub/master/docs/imgs/create-policy.png"/>
+</p>
+
+Next, select a *type* for the Policy. There are 2 types of Access Policy within DataHub:
```

Comment:
This is confusing:

> Step 1. Choose Policy Type
> In the first step, we'll give our Policy a name and description

Consider splitting out Name & Description as Step 1 & providing some guidance around the naming conventions. What are common naming patterns we recommend?

Then, move on to Step 2: Choose Policy Type